bundles/org.openhab.binding.awattar/src/main/resources/OH-INF/i18n/awattar.properties

```diff
@@ -12,6 +12,8 @@ bridge-type.config.awattar.bridge.basePrice.label = Base price
 bridge-type.config.awattar.bridge.basePrice.description = Specifies the net base price per kWh
 bridge-type.config.awattar.bridge.timeZone.label = Time zone
 bridge-type.config.awattar.bridge.timeZone.description = Time zone to apply to the hour definitions. Default CET aligns to the aWATTar API
+brifge-type.config.awattar.bridge.serviceFee.label = Service fee
```

Comment:
Typo "bridge".

Reply:
No need to wait for fixing this ypo. :-)

Reply:
I just put that there to remember.. I'm case ther might be more changes needed... :)